src/components/VmDetails/cards/DetailsCard.js

```diff
@@ -1,39 +1,258 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
 
+import { generateUnique } from '../../../helpers'
```

Comment:
I don't really like a lot of `../`, can we make helpers, actions, and other basic modules as alias in webpack? Not in this PR, but maybe in some next.

Reply:
Maybe, but really only if vs-code will still be able to code-assist etc over the webpack alias.